inc/vite.php

```
@@ -30,9 +30,40 @@ private function load_manifest() {
             $this->dist_path . '/manifest.json'
         ];
 
+        $cache_key = 'djz_vite_manifest_v2';
```

PHPMD (2.15.0)

The variable $cache_key is not named in camelCase. (undefined)

(CamelCaseVariableName)